app/src/androidTest/java/com/veniosg/dir/test/acceptance/FileManagerActivityTest.java

```diff
@@ -38,6 +40,17 @@
     private final File testChildDirectory = new File(testDirectory, "testChildDir");
     private final File testCopyDestination = new File(testDirectory, "testCopyDestination");
     private final File testChildFile = new File(testDirectory, "testChildFile");
+    private final File testDirectoryWithZeroItems = new File(sdCardDir, "testDirWithZeroItems");
```

Comment:
Please create these below testDirectory. That way you need nothing more in tearDown, everything should be cleaned up by `cleanDirectory(testDirectory)`

Reply:
Done!